server/src/services/data_queries.service.ts

```diff
@@ -31,7 +31,7 @@ export class DataQueriesService {
 
     return await this.dataQueriesRepository.find({
       where: whereClause,
-      order: { name: 'ASC' },
+      order: { createdAt: 'ASC' },
```

Comment:
Will it looks good if its in descending order?
Latest query at top?
@Navaneeth-pk

Reply:
Yes 👍

Reply:
Done